sdk/samples/iot/paho_iot_hub_pnp_sample.c

```diff
@@ -852,18 +852,19 @@ static az_result invoke_getMaxMinReport(
       command_start_time_value_buffer,
       sizeof(command_start_time_value_buffer),
       &incoming_since_value_len));
-  az_span start_time_span
-      = az_span_create((uint8_t*)command_start_time_value_buffer, incoming_since_value_len);
-
-  IOT_SAMPLE_LOG_AZ_SPAN("start time:", start_time_span);
```

Comment:
@momuno - The span creation and logging should happen after the `start_time_span` size check, correct?

Reply:
This looks correct.  @danewalton thoughts?

Reply:
👍